src/Helpers/ContextHelper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Helpers;
+
+use Illuminate\Support\Str;
+
+class ContextHelper
+{
+    public static function getHelper(string $helper)
+    {
+        $helperClass = match($helper) {
+            "array" => config("blt.helpers.array"),
```

Comment:
```suggestion
            "array" => config("blt.helpers.array") ?? ArrayHelper::class,
```

And please change all of them this way.